processor/src/main/java/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMapsWithNullCheck.java

```diff
@@ -63,6 +63,9 @@ public SetterWrapperForCollectionsAndMapsWithNullCheck(Assignment decoratedAssig
         if (isDirectAssignment() || getSourcePresenceCheckerReference() == null ) {
             imported.addAll( getNullCheckLocalVarType().getImportTypes() );
         }
+        if ( isMapNullToDefault() ) {
+            imported.add( targetType.getImplementationType() );
```

Comment:
In line 52 is a check if `targetType.getImplementationType()` is not `null` - so could it also be the case that at this point it is `null`? In this case we also need the check here to prevent adding `null` to this `Set`.

Reply:
Good point. Will fix. Thanks